### DISCUSSION TITLE
Welcome to security-and-auditing-full-course-s23 Discussions!

### DISCUSSION THREAD

**Discussion by PatrickAlphaC** [Cyfrin/security-and-auditing-full-course-s23#1] [Announcements]
## Welcome!

Course coming soon....

**Comment by iftikharuddin** [comment]
Finally I received the code, excited for this!

**Reply by PatrickAlphaC** [reply]
yyeesss!!

**Comment by Simon-Busch** [comment]
Can't wait to receive my code 🥹🚀

**Comment by fdicarlo** [comment]
Hello everyone, glad to be here :)

**Comment by Isaaco3349** [comment]
finally, i got access to the course lfg !!!!

**Comment by garyghayrat** [comment]
So excited to get started on this! 🥳

**Comment by Im-FEMedina** [comment]
Greetings to everyone.  Looking forward towards this Lewis and Clark journey.  Using this metaphor, does this make Patrick Sacagawea? Anyways, ought to be a great life long experience.

**Comment by 0502lian** [comment]
I am incredible exited to get here!

**Comment by Mucunguzi256** [comment]
Hey,
I need an access code to the intial courses. Very much interested in this course.

**Reply by PatrickAlphaC** [reply]
No code needed! 

https://updraft.cyfrin.io/courses/security
